less/home-page.less

```diff
@@ -24,8 +29,29 @@
       margin: 20px 2.5%;
       border-radius: 20px;
       height: 100%;
+
+      }
+
+      .blast {
+        @media @mobile {
+          width: 90%;
+          margin: 20px auto;
+        }
+      }
+
+      .sky {
+        @media @mobile {
+          display: none;
+        }
+      }
+
+      .launch {
+        @media @mobile {
+          display: none;
+        }
+      }
```

Comment:
Nit-picking here because your code looks beautiful - you could select both of these classes on the same line by separating them by a comma